bin/epaa.py

```diff
@@ -29,7 +29,7 @@
 from string import Template
 
 __author__ = 'Christopher Mohr'
-VERSION = "1.0"
+VERSION = "1.1"
```

Comment:
not sure, does this script need its own version?

Reply:
I guess it's not necessary but since it always contained a version I thought it would make sense to increase the version since a couple of things have been changed.